packages/cardpay-sdk/sdk/scheduled-payment-module.ts

```diff
@@ -1289,13 +1290,26 @@ export default class ScheduledPaymentModule {
     });
     let gasStationResponse = await getCurrentGasPrice(chainId);
```

Comment:
I think `gasPricesInNativeWei` would be a more informative name for this variable

Reply:
updated